### ISSUE TITLE
Porting the extension to Firefox

### ISSUE THREAD

**Issue by Cpt-T-Krik** [Cpt-T-Krik/rbtv_mission_control#2]
Firefox is still a widely used browser. Porting the extension to Firefox is a very important step to reach more users. I hope some of you guys can work on this issue.

**Comment by HYMXDEV** [comment]
I will try to port it to Firefox today. It will be my first Firefox extension, but I read the documentation of Firefox addons and it seems to be not so much work.

**Comment by HYMXDEV** [comment]
I tried to create a Firefox extension and after some changes most things are working, but at the moment I have problems with Firefox´s same origin policy because it is more difficult then Chrome´s. As a workaround I am using my server as proxy, but I am working on the problem.
![rbtv_mc_firefox_port](https://cloud.githubusercontent.com/assets/12176022/7419647/be441ad2-ef76-11e4-8857-9cdc062f32a7.jpg)

**Comment by HYMXDEV** [comment]
I just uploaded the Firefox extension as a fork of yours:
https://github.com/HYMXDEV/rbtv_mission_control/tree/master
Everything should work fine.

**Comment by Cpt-T-Krik** [comment]
You fixed the problem with the Firefox´s same origin policy? 
I'll test the extension. After the approval it will be added to the repo.

**Comment by HYMXDEV** [comment]
Yes it is fixed

**Comment by Cpt-T-Krik** [comment]
The functionalities are working great. Great work! But unfortunately there seems to be problem with the font style of the current version of the Firefox extension. Please see the screenshot below: 

![rbtv_mc_firefox_port_font_one](http://s4.postimg.org/jnw063w7x/Bildschirmfoto_2015_05_01_um_14_34_51.png)

Also the opener image in the information page is missing.

![rbtv_mc_firefox_port_opener](http://s24.postimg.org/9vd9nv9t1/Bildschirmfoto_2015_05_01_um_14_34_37.png)

Please fix these two issues. After that I'll add it to the main repo of the project.

**Comment by HYMXDEV** [comment]
I fixed the missing image but i couldnt reproduce your font problem. Which Firefox version do you use?
![rbtv_mc_firefox](https://cloud.githubusercontent.com/assets/12176022/7430413/274d12bc-f013-11e4-9513-b9712622c86a.jpg)

**Comment by Cpt-T-Krik** [comment]
I'm using Firefox version 37.0.2 on Mac OS X 10.10.3.

**Comment by HYMXDEV** [comment]
Okay it's definite a Mac OS problem. On all my windows mashines the font is displayed correctly, but my Mac show the same problem as yours. At the moment I don't know why the font is different, but I'm searching...

**Comment by HYMXDEV** [comment]
Okay it's definite a Mac OS problem. On all my windows machines the font is displayed correctly, but my Mac shows the same problem as yours. At the moment I don't know why the font is different, but I'm searching...

# Edit:
There is still a difference between Mac and PC, but for now it's the best I could do.

![rbtv_mc_firefox](https://cloud.githubusercontent.com/assets/12176022/7433582/39a37170-f033-11e4-8ca3-b88ab4725fbf.jpg)

**Comment by Cpt-T-Krik** [comment]
Looks great! The difference doesn't really matter. It's an issue which can be fixed in the future. 
I'll add your Firefox port of this extension to the main repo and will upload it to the Firefox-Add-on page, if you don't mind. Are you also on reddit? If so, please let me know, so I can add your name to the credits!

**Comment by HYMXDEV** [comment]
No, I of course I don't mind. My reddit name also is [HYMXDEV](https://www.reddit.com/u/HYMXDEV/).

**Comment by Cpt-T-Krik** [comment]
You should check the following: 

![FF test](http://s10.postimg.org/jgt0b8rvt/Bildschirmfoto_2015_05_04_um_17_50_26.png)
